tldw_Server_API/app/api/v1/endpoints/connectors.py

```
@@ -114,27 +114,27 @@ async def oauth_callback(
+        pol = await get_policy(db, org_id)
+        if not pol:
+            pol = get_default_policy_from_env(org_id)
+        ok, why = evaluate_policy_constraints(
+            pol,
+            provider=provider,
+            remote_path=None,
+            notion_workspace_id=notion_workspace_id,
+            account_email=acct_email,
+        )
+        if not ok:
+            raise HTTPException(status_code=403, detail=why or "Account not permitted by org policy")
```

Ruff (0.14.6)

Abstract `raise` to an inner function

(TRY301)